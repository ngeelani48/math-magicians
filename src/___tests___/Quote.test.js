import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../components/Quote';

it('check if the Quote section renders properly', () => {
  const tree = renderer.create(<Quotes />).toJSON();
  expect(tree).toMatchSnapshot();
});
