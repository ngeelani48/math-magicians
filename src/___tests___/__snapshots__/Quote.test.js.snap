// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check if the  section renders properly 1`] = `
<div
  className="quotes"
>
  <h2>
    <u>
      Quote
    </u>
  </h2>
  <div>
    <p />
    <p>
      <b />
    </p>
  </div>
</div>
`;

exports[`check if the Home section renders properly 1`] = `
<div
  className="quotes"
>
  <h2>
    <u>
      Quote
    </u>
  </h2>
  <div>
    <p />
    <p>
      <b />
    </p>
  </div>
</div>
`;

exports[`check if the Quote section renders properly 1`] = `
<div
  className="quotes"
>
  <h2>
    <u>
      Quote
    </u>
  </h2>
  <div>
    <p />
    <p>
      <b />
    </p>
  </div>
</div>
`;
